packages/ui/src/components/Channel/Scroller.tsx

```diff
@@ -396,14 +424,20 @@ const Scroller = forwardRef(
             ref={flatListRef as React.RefObject<Animated.FlatList<db.Post>>}
             // This is needed so that we can force a refresh of the list when
             // we need to switch from 1 to 2 columns or vice versa.
-            key={channelType}
+            key={channel.type}
             data={postsWithNeighbors}
             renderItem={listRenderItem}
             ListEmptyComponent={renderEmptyComponent}
             keyExtractor={getPostId}
             keyboardDismissMode="on-drag"
             contentContainerStyle={contentContainerStyle}
-            columnWrapperStyle={channelType === 'gallery' && columnWrapperStyle}
+            columnWrapperStyle={
+              // FlatList raises an error if `columnWrapperStyle` is provided
+              // with numColumns=1, even if the style is empty
```

Comment:
tbc, this comment was relevant before these changes – I didn't get why we were checking `channelType === 'gallery'` here until I tried it out